src/authenticated-app.tsx

```diff
@@ -3,18 +3,24 @@ import { Switch, Route } from 'react-router-dom'
 
 import { ProjectTypeSelection } from 'screens/project-type-selection'
 // import { SmallBuildQuoteCalculator } from 'screens/small-build-quote-calculator'
-import { Materials } from 'screens/materials'
-import { Wages } from 'screens/wages'
-import { Rentals } from 'screens/rentals'
+import { Materials } from 'screens/materials/materials'
+import { Wages } from 'screens/wages/wages'
+import { Rentals } from 'screens/rentals/rentals'
+import { MaterialsEditor } from 'screens/materials/materials-editor'
+import { WagesEditor } from 'screens/wages/wages-editor'
+import { RentalsEditor } from 'screens/rentals/rentals-editor'
```

Comment:
This folder should be one level deep imo

Reply:
Ok, I changed it back.